crates/rpc-types/src/eth/transaction/optimism.rs

```diff
@@ -30,8 +30,8 @@ pub struct OptimismTransactionReceiptFields {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub l1_fee: Option<U256>,
     /// L1 fee scalar for the transaction
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub l1_fee_scalar: Option<U256>,
+    #[serde(default, skip_serializing_if = "Option::is_none", with = "l1_fee_scalar_serde")]
```

Comment:
```suggestion
    #[serde(skip_serializing_if = "Option::is_none", with = "l1_fee_scalar_serde")]
```

Reply:
Without the serde(default) the deserialization works when the field is Value::Null, but fails when it's not set.
https://stackoverflow.com/a/44303505

```
thread 'eth::transaction::optimism::tests::deserialize_l1_fee_scalar' panicked at crates/rpc-types/src/eth/transaction/optimism.rs:126:88:
called `Result::unwrap()` on an `Err` value: Error("missing field `l1FeeScalar`", line: 0, column: 0)

```

Should I remove it ?

Reply:
TIL